packages/langium/src/services.ts

```diff
@@ -160,11 +162,13 @@ export type LangiumDefaultSharedServices = {
     ServiceRegistry: ServiceRegistry
     lsp: {
```

Comment:
Idea: Order by name? Then, we can find services easier. Don't know if that makes sense...